### PR TITLE
Update rsyslog to also listen on TCP port 515

### PR DESCRIPTION
**Pull request by Kami**
Right now long messages are getting truncated.

I still need to update st2.conf to utilize the TCP transport.

### CODE REVIEW

File: modules/profile/manifests/rsyslog.pp [hunk 1]
```diff
@@ -12,6 +12,18 @@
     notify => Service['rsyslog']
   }
 
+  # Note: We also listen on TCP so long messages are notr truncated
```
Comment (manasdk) [inline modules/profile/manifests/rsyslog.pp]: notr -> not

**Comment by Kami** [comment]
Where should I put `ini_setting` directive which configures st2.conf to use TCP transport + port 514, st2server.pp?

**Comment by DoriftoShoes** [comment]
https://github.com/StackStorm/st2workroom/blob/master/modules/profile/manifests/st2server.pp
In there somewhere.

**Comment by Kami** [comment]
Tested the changes, seem to be working fine.

**Comment by jfryman** [comment]
lgtm

**Pull request merged**